packages/forms/src/directives/radio_control_value_accessor.ts

```diff
@@ -154,6 +157,9 @@ export class RadioControlValueAccessor extends BuiltInControlValueAccessor imple
    */
   @Input() value: any;
 
+  private callSetDisabledState: SetDisabledStateOption =
```

Comment:
Do we need explicit `SetDisabledStateOption` (I'm wondering if it's inferred)?
```suggestion
  private callSetDisabledState: SetDisabledStateOption =
```

Reply:
Indeed we do not.